packages/hydrogen/src/cart/cartSetIdDefault.test.ts

```diff
@@ -6,19 +6,15 @@ describe('cartSetIdDefault', () => {
     const setCartId = cartSetIdDefault();
     const headers = setCartId('gid://shopify/Cart/c1-123');
 
-    expectTypeOf(setCartId).toEqualTypeOf<
-      (cartId: string, headers: Headers) => void
-    >;
+    expectTypeOf(setCartId).toEqualTypeOf<(cartId: string) => Headers>;
```

Comment:
@wizardlyhel  I checked the type returned by `cartSetIdDefault` and it's `(cartId: string) => Headers`. I've no idea why this was passing before either. Should be ok, right?